airflow/providers/google/cloud/operators/bigquery.py

```diff
@@ -116,6 +119,9 @@ def get_link(
         if len(job_ids) < self.index:
             return None
         job_id = job_ids[self.index]
+        # converts project_id:location.job_id to project_id:location:job_id
+        if job_id:
+            job_id = str(job_id).replace(".", ":")
```

Comment:
Instead of repeating this logic twice - I think that it's worth extracting it to a helper static method in the hook

Reply:
the problem does not currently exist